### PR TITLE
Feature/add environment display

### PR DESCRIPTION
**Pull request by solidgoldpig**
Also bump dependencies

### CODE REVIEW

**Comment by solidgoldpig** [comment]
Shows PLATFORM_ENV and DEPLOYMENT_ENV 

GOV_UK_-_The_best_place_to_find_government_services_and_information

- Platform env only shown if it's test or integration
- Deployment env only shown if it's dev or staging

File: templates/nunjucks/_base.njk.html [hunk 1]
```diff
@@ -10,6 +9,31 @@
...
+  transform: rotate(90deg);
+  margin-left: 30px;
+  margin-top: 0px;
+  height: 30px;
+  overflow: hidden;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+</style>
+<div class="environmentDisplay">
```
Comment (umaar) [inline templates/nunjucks/_base.njk.html]: minor: our convention is hyphenation I think

**Review by umaar — APPROVED** [review]
Looks really good

File: templates/nunjucks/_base.njk.html (hunk 1 above)
Comment (solidgoldpig) [reply templates/nunjucks/_base.njk.html]: Sorted

**Pull request merged**